### PR TITLE
Implemented initial EE test - to verify basic UI functions

### PR DESCRIPTION
**Pull request by ldimaggi**

### CODE REVIEW

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 4c68e0922481e67b2254da12fc7d7d0a901c2f57
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 8724d6ed739db68ff2e6f3169295f4ebf3785a4e
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c904ec8ed83a567406d67d80f2a81259d7c48cd7
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by ldimaggi** [comment]
[test]

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: ad7e1b0bcd2aa6d8603b1914a34223f70f3c9afb
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: eed454a05466dcdc9fb75090c068d2cf072a9ba9
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by ldimaggi** [comment]
[test]

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: ad7e1b0bcd2aa6d8603b1914a34223f70f3c9afb
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

 * Commit: be0e7ffb05a41661d18db83507ca1e6d3b281057
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: 11921a3182c9b6163cd7c375867a5ac0641a24ff
  * Your subject line is longer than 50 characters

 * Commit: 1cffcfa82094d2ad7be1f96273952ee8aa538467
  * Your subject line is longer than 50 characters

 * Commit: 35cd8251eae8099d68b92833c68553931179d7d7
  * Your subject line is longer than 50 characters

 * Commit: 46958b426631a7f702ea0c239d656a8ffe4e77d7
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

 * Commit: 1b40bcf7d9a3e808d5e2780699554cd03a30bbb9
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

 * Commit: eed454a05466dcdc9fb75090c068d2cf072a9ba9
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

File: Dockerfile.builder [hunk 1]
```diff
@@ -21,25 +20,45 @@ RUN set -ex \
   done
 
 #ENV NPM_CONFIG_LOGLEVEL info
-ENV NODE_VERSION 6.5.0
+ENV NODE_VERSION 6.9.2
```
Comment (joshuawilson) [inline Dockerfile.builder]: Why is this changing? @kbsingh did you authorize this?

File: package.json [hunk 2]
```diff
@@ -108,7 +108,7 @@
     "angular2-oauth2": "1.3.10",
     "chunk-manifest-webpack2-plugin": "1.0.1",
     "core-js": "2.4.1",
-    "fabric8-planner": "0.22.9",
+    "fabric8-planner": "0.22.5",
```
Comment (joshuawilson) [inline package.json]: please don't change this

**Comment by joshuawilson** [comment]
@ldimaggi it looks like you don't have a clean merge. I'll help you fix it.

**Comment by ldimaggi** [comment]
[test]

File: Dockerfile.builder (hunk 1 above)
Comment (kbsingh) [inline Dockerfile.builder]: my bad, i did a rebase to master, and that pulled in these changes. @ldimaggi you should be ok to drop all this and only cherry pick the required 2 commits

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 7e432b9bdc70903c19b9c3cd4a11a664955e5a2a
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

 * Commit: 4e4aadc119da3f1e317592fa66f1f11fead62f89
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

 * Commit: bc8f6ef7df1cdf3ef44de90529b2dc764cd104c1
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

File: Dockerfile.builder (hunk 1 above)
Comment (joshuawilson) [inline Dockerfile.builder]: @ldimaggi this fix didn't make it through the rebasing. Please change this back to 6.5.0.

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 19dc6977353e5c4926203b88e15d9ce347ac4d50
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by rawlingsj** [comment]
@joshuawilson @ldimaggi  the error shows an issue building the snapshot UI docker image.  So AFAICS the error is valid and if merged will break the release of fabric8-ui.  There is a missing `nginx.conf` file from this pull request.  I also noticed lots changes to dockerfiles and references to planner - is that expected?  
```
Digest: sha256:74fc0d9fd32e34f5042c6f6a7f65ec63e1837cdc83459b4552f78a10f0eb61ed
Status: Downloaded newer image for registry.centos.org/kbsingh/openshift-nginx:latest
 ---> d935ef3f09a9
Step 2 : ENV LANG en_US.utf8
 ---> Running in 9772591aae99
 ---> d93f266704ee
Removing intermediate container 9772591aae99
Step 3 : USER root
 ---> Running in 84fd26608bd0
/home/jenkins/workspace/-projects_fabric8-ui_PR-585-5FSMZROIKWRVEKR2NOY35S742ARO6N4UBKJAMCM6ZYOXU4GX42TA@2 # exit
 ---> 52be1e52825d
Removing intermediate container 84fd26608bd0
Step 4 : ADD nginx.conf /etc/nginx/nginx.conf
lstat nginx.conf: no such file or directory
```

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c9bd4e7a0b9289ed296241f64a806b7747bd403e
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: c9bd4e7a0b9289ed296241f64a806b7747bd403e
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

 * Commit: f37d35de05843d13de0d58097cdb924ff664b8bb
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by ldimaggi** [comment]
[test]

**Comment by fabric8cd** [comment]
@ldimaggi snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-585.int.rdu2c.fabric8.io

**Comment by GitCop** [comment]
There were the following issues with your Pull Request

 * Commit: 5a6213ee93d32dbf6ba22e03f1a42f03df280f48
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**
  * Your subject line is longer than 50 characters

 * Commit: 67ae54434aa23cf5fe51325f3c86cad4ef24103e
  * Commits must be in the following format: **%{type}(%{scope}): %{description}**

Guidelines are available at https://github.com/fabric8io/fabric8-ui

---
This message was auto-generated by https://gitcop.com

**Comment by fabric8cd** [comment]
@ldimaggi snapshot fabric8-ui is deployed and available for testing at https://fabric8-ui-fabric8-ui-pr-585.int.rdu2c.fabric8.io

**Pull request merged**